hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/routes/BaseRoute.java

```diff
@@ -16,8 +16,7 @@
 public abstract class BaseRoute extends RouteBuilder {
 	protected static final String JMS_DESTINATION_NAME_FORMAT = "queue:///%s?targetClient=1&&mdWriteEnabled=true";
 	
-	protected static final String MQ_URL_FORMAT = "jmsComponent:queue:%s?exchangePattern=InOut&replyTo=queue:///%s&replyToType=Shared&allowAdditionalHeaders=JMS_IBM_MQMD_MsgId";
-	
+	protected static final String MQ_URL_FORMAT = "jmsComponent:queue:%s?exchangePattern=InOut&replyTo=queue:///%s&replyToType=Exclusive&allowAdditionalHeaders=JMS_IBM_MQMD_MsgId&deliveryMode=1&requestTimeout=7000&explicitQosEnabled=true&timeToLive=7000";
```

Comment:
We probably need to start adding some comments about why we are using certain properties and their values here for properties like requestTimeout, timeToLive. Or a general about the expected endpoint behaviour. I can add comment on replyToType of 'Shared' being required as multiple apps could be connecting to the same queue and this indicates Camel needs to be selective about which messages it attempts to retrieve.